### ISSUE TITLE
Doesn't load no image facebook posts

### ISSUE THREAD

**Issue by goldenskygiang** [wasteam/waslibs#19]

**Comment by javitosanchez** [comment]
Thanks for reporting this. We are investigating this issue. We'll keep you updated.

**Comment by javitosanchez** [comment]
We are not able to reproduce it. Could you send us a repro?

**Comment by goldenskygiang** [comment]
As you can see that, Facebook posts with images can be loaded in my app.
![screenshot 55](https://cloud.githubusercontent.com/assets/10509643/13569820/dd01a6a8-e49c-11e5-8d81-ea96effd2104.png)
But in the image below, I show you the time when the selected posts are posted both in my app and in the Facebook page.
The selected post in the browser was posted between these two posts in the app
You can see the post in the browser doesn't have an image, and the app doesn't show it.
![screenshot 56](https://cloud.githubusercontent.com/assets/10509643/13569857/30c84d78-e49d-11e5-9a8a-d01ab6ce0738.png)
So in conclusion, if a Facebook post has an image or a link, it can be shown in my app, but other posts aren't shown. 
So can you help me fix this bug?
(Sorry because of my bad English !)

**Comment by javitosanchez** [comment]
Solved in this version. Thanks for reporting this.
